server/views/challenges/showBonfire.jade

```diff
@@ -30,15 +30,15 @@ block content
                                         ul: li: a(href=""+link, target="_blank") !{MDNkeys[index]}
                             .button-spacer
                 if (user)
-                label.btn.btn-primary.btn-block.btn-lg#submitButton
+                button.btn.btn-primary.btn-block.btn-lg#submitButton
                         | Run tests (ctrl + enter)
                 .button-spacer
                 .btn-group.input-group.btn-group-justified
-                    label.btn.btn-primary.btn-primary-ghost.btn-lg#trigger-reset-modal
+                    a.btn.btn-primary.btn-primary-ghost.btn-lg#trigger-reset-modal(href="#" role="button")
```

Comment:
`button` does not go along with `display:table-cell`, thus `a` which can receive `focus` and does not mess up the style.
added `role="button"`, so that its is semantically exposed as `button` to assistive technology.